lib/vehicle.rb

```diff
@@ -22,4 +30,16 @@ def antique?
   def electric_vehicle?
     @engine == :ev
   end
+
+  def plate_type
+    if registration_date != nil
+      if antique?
```

Comment:
Again, moving forward, try to avoid nested if statements as they are process heavy. We can usually refactor this using our boolean operators.